### DISCUSSION TITLE
eth withdrawal from dbk chain to my wallet does not arrive after 12 days

### DISCUSSION THREAD

**Discussion by jmjurado77** [blockscout/blockscout#10581] [Support]
Good afternoon,
On July 29th I started a withdrawal of 0.2 eth from the dbk chain to my wallet. When I started it, I was warned that the transaction could take around 7 days, but I'm starting to get very worried because it's been 12 days and I still haven't received my eth. I need you to confirm what's going on.
![trasnferencia dbkchain](https://github.com/user-attachments/assets/a8764cb3-4d45-4fc5-a057-097bce469974)

**Comment by ekawibisono** [comment]
Hello @jmjurado77 , for this problem, please ask the bridge or platform you are using. This is github blockscout.

Thanks
